src/main/kotlin/org/jitsi/nlj/transform/node/incoming/Vp8Parser.kt

```diff
@@ -44,6 +44,10 @@ class Vp8Parser(
     // Stats
     private var numKeyframes: Int = 0
 
+    private val pictureIdState = StateChangeLogger("picture id", logger)
+    private val extendedPictureIdState = StateChangeLogger("extended picture ID", logger)
+    private val tl0PicIdxWithoutTidState = StateChangeLogger("TID without TL0PICIDX", logger)
```

Comment:
The log messages this produces are quite cryptic:
`Packet $instance does not have TID without TL0PICIDX` and 
`Packet $instance has TID without TL0PICIDX when source previously did not.`.

You could log something like "Packet has $desc" and "Packet source no longer has $desc", and initialize $desc to "missing picture id" and "TID with missing TL0PICIDX".

Reply:
Also, the variable name is flipped: should be `tidWithoutTl0PicIdx`